api/envoy/extensions/filters/http/lua/v3/lua.proto

```diff
@@ -78,6 +80,13 @@ message Lua {
   //         stat_prefix: bar_script # This emits lua.bar_script.errors etc.
   //
   string stat_prefix = 4;
+
+  // Determines if HTTP body is buffered when Lua script executes httpCall.
+  // If the setting is false, Envoy will put pressure on the downstream client to stop sending
+  // data. If the setting is true, Envoy will try to buffer entire body sent by downstream client,
+  // but buffering may exceed maximum buffer limit per connection and the connection may be dropped.
+  // Defaults to true.
+  google.protobuf.BoolValue buffer_body = 5;
```

Comment:
I am fine to this feature. But the naming `buffer_body` will indicates that this field is used to determine whether the lua filter will buffer the body data.

So, I think `flow_backpressure` or `backpressure` maybe better.

Reply:
Renamed to `flow_control`. This was great suggestion. Thanks!